crates/cheatnet/tests/cheatcodes/mod.rs

```diff
@@ -9,4 +13,13 @@ mod precalculate_address;
 mod roll;
 mod spoof;
 mod spy_events;
+mod store;
 mod warp;
+
+pub fn map_entry_address(var_name: &str, key: &[Felt252]) -> Felt252 {
+    calculate_variable_address(felt_selector_from_name(var_name), Some(key))
+}
+
+pub fn variable_address(var_name: &str) -> Felt252 {
+    calculate_variable_address(felt_selector_from_name(var_name), None)
+}
```

Comment:
We won't be calculating the selector under the hood since we need it from the user. Right now there isn't a reliable option covering all cases which will work for this interface (long strings only supported in macros)